### PR TITLE
[zh-cn]: create the translation of the MathML global attribute `mathsize`

### PR DESCRIPTION
**Pull request by tianyeeT**
### Related issues and pull requests
#20746

### CODE REVIEW

File: files/zh-cn/web/mathml/global_attributes/mathsize/index.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+---
+title: mathsize
+slug: Web/MathML/Global_attributes/mathsize
+l10n:
+  sourceCommit: 07f0cf4375aaa02e1071d8bd0e8518db7609b7a9
+---
+
+{{MathMLRef}}{{Deprecated_Header}}
+
+**`mathbackground`** [全局属性](/zh-CN/docs/Web/MathML/Global_attributes) 设置 MathML 元素的[字体大小](/zh-CN/docs/Web/CSS/color)。
```
Comment (JasonLamv-t) [inline files/zh-cn/web/mathml/global_attributes/mathsize/index.md:10]:
```suggestion
**`mathbackground`** [全局属性](/zh-CN/docs/Web/MathML/Global_attributes) 设置 MathML 元素的[字体大小](/zh-CN/docs/Web/CSS/font-size)。
```

File: files/zh-cn/web/mathml/global_attributes/mathsize/index.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+> **备注：** 尽可能使用 CSS 对 MathML 进行样式设置。`mathsize` 属性应仅在不支持 CSS 并且将被 CSS `font-size` 属性值覆盖的应用程序中包含。
```
Comment (JasonLamv-t) [inline files/zh-cn/web/mathml/global_attributes/mathsize/index.md:12]:
```suggestion
> **备注：** 尽可能使用 CSS 对 MathML 进行样式设置。`mathsize` 属性只应包含在不支持 CSS 的应用程序中，并且如果设置的话，其将被 CSS `font-size` 属性值覆盖。
```

File: files/zh-cn/web/mathml/global_attributes/mathsize/index.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+```html-nolint
+<!-- <length> 值 -->
+<math mathsize="12px">
+<math mathsize="0.8em">
+
+<!-- <percentage> 值 -->
+<math mathsize="80%">
+```
+
+### 可选值
```
Comment (JasonLamv-t) [inline files/zh-cn/web/mathml/global_attributes/mathsize/index.md:25]:
```suggestion
### 值
```

File: files/zh-cn/web/mathml/global_attributes/mathsize/index.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+- {{cssxref("&lt;length&gt;")}}
+
+  - : 一个正值的 {{cssxref("&lt;length&gt;")}}。对于大多数与字体相关的单位（例如 `em` 和 `ex`），字体大小是相对于父元素的字体大小的。
+
+- {{cssxref("&lt;percentage&gt;")}}
+
+  - : 一个正值的 {{cssxref("&lt;percentage&gt;")}}，相对于父元素的字体大小。
```
Comment (JasonLamv-t) [inline files/zh-cn/web/mathml/global_attributes/mathsize/index.md:33]:
```suggestion
- {{cssxref("&lt;length&gt;")}}

  - : 一个正的 {{cssxref("&lt;length&gt;")}} 值。对于大多数与字体相关的单位（例如 `em` 和 `ex`），字体大小是相对于父元素的字体大小的。

- {{cssxref("&lt;percentage&gt;")}}

  - : 一个正的 {{cssxref("&lt;percentage&gt;")}} 值，相对于父元素的字体大小。
```

File: files/zh-cn/web/mathml/global_attributes/mathsize/index.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+
+> **备注：** 一些浏览器可能还接受[旧版 MathML 长度](/zh-CN/docs/Web/MathML/Values#legacy_mathml_lengths)。
```
Comment (JasonLamv-t) [inline files/zh-cn/web/mathml/global_attributes/mathsize/index.md:35]:
```suggestion
> **备注：** 一些浏览器可能还接受[旧版 MathML 长度](/zh-CN/docs/Web/MathML/Values#旧版_mathml_长度)。
```

File: files/zh-cn/web/mathml/global_attributes/mathsize/index.md [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+
+## 规范
+
+{{Specifications}}
+
+- 在 MathML 3 和更早版本中，支持关键字 `small`、`normal` 和 `big`，以及 MathML3 特定的长度语法。。自 MathML 核心版本以来，语法与 CSS {{cssxref("&lt;length-percentage&gt;")}} 值相匹配。
```
Comment (JasonLamv-t) [inline files/zh-cn/web/mathml/global_attributes/mathsize/index.md:41]:
```suggestion
- 在 MathML 3 和更早版本中，支持关键字 `small`、`normal` 和 `big`，以及 MathML3 特定的长度语法。自 MathML 核心版本以来，语法与 CSS {{cssxref("&lt;length-percentage&gt;")}} 值相匹配。
```

File: files/zh-cn/web/mathml/global_attributes/mathsize/index.md [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+
+- 此属性设计用于不支持 CSS 的 MathML 应用程序。自 MathML 核心版本以来，推荐使用等效的 CSS。
+
+## 浏览器兼容性
+
+{{Compat}}
+
+## 参见
+
+- 所有 [全局属性](/zh-CN/docs/Web/MathML/Global_attributes)
```
Comment (JasonLamv-t) [inline files/zh-cn/web/mathml/global_attributes/mathsize/index.md:51]:
```suggestion
- 所有[全局属性](/zh-CN/docs/Web/MathML/Global_attributes)
```

**Review by JasonLamv-t — DISMISSED** [review]
Some suggestions.

File: files/zh-cn/web/mathml/global_attributes/mathsize/index.md [hunk 8]
```diff
@@ -0,0 +1,52 @@
+---
+title: mathsize
+slug: Web/MathML/Global_attributes/mathsize
+l10n:
+  sourceCommit: 07f0cf4375aaa02e1071d8bd0e8518db7609b7a9
+---
+
+{{MathMLRef}}{{Deprecated_Header}}
+
+**`mathbackground`** [全局属性](/zh-CN/docs/Web/MathML/Global_attributes) 设置 MathML 元素的[字体大小](/zh-CN/docs/Web/CSS/font-size)。
```
Comment (yin1999) [inline files/zh-cn/web/mathml/global_attributes/mathsize/index.md:10]:
```suggestion
**`mathsize`** [全局属性](/zh-CN/docs/Web/MathML/Global_attributes)设置 MathML 元素的[字体大小](/zh-CN/docs/Web/CSS/font-size)。
```

**Review by yin1999 — APPROVED** [review]
:+1:

**Pull request merged**